fe1-web/src/features/evoting/objects/Feature.ts

```diff
@@ -0,0 +1,29 @@
+import { Hash, PublicKey } from 'core/objects';
+
+export namespace EvotingFeature {
+  export interface Lao {
+    id: Hash;
+    organizer: PublicKey;
+    witnesses: PublicKey[];
+  }
+
+  export interface EventState {
+    readonly eventType: string;
+
+    readonly id: string;
+
+    readonly idAlias?: string;
+
+    readonly name: string;
+
+    readonly start: number;
+
+    readonly end?: number;
+  }
+
+  export interface Event {
+    readonly id: Hash;
+
+    toState(): EventState;
+  }
```

Comment:
we may not need this re-declaration (later on...), for now, I agree that it's a minor evil 😄